python/ray/experimental/sgd/pytorch/pytorch_distributed_sgd.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import torch
+
+import ray
+
+from ray.experimental.sgd.pytorch.pytorch_runner import PyTorchRunner
+from ray.experimental.sgd.pytorch import utils
+
+
+class PyTorchDistributedSGD(object):
+    """Train a model using distributed PyTorch.
+
+    Args:
+        model_creator (dict -> torch.nn.Module): creates the model using the
+            config.
+        data_creator (dict -> Dataset, Dataset): creates the training and
+            validation data sets using the config.
+        optimizer_creator (model, dict -> loss, optimizer): creates the loss
+            and optimizer using the config.
+        config (dict): configuration passed to 'model_creator', 'data_creator',
+            and 'optimizer_creator'.
+        batch_size (int): batch size used for SGD.
+        backend (string): backend used for distributed SGD. "gloo" or "nccl".
+        mixed_precision (bool): ???
```

Comment:
Let's remove this for now - maybe we can have an issue opened to track some features